.expeditor/verify.pipeline.yml

```diff
@@ -33,6 +33,15 @@ steps:
       docker:
         image: ruby:2.6-stretch
 
+- label: run-lint-and-specs-ruby-2.7
+  command:
+    - bundle install --jobs=7 --retry=3 --without docs debug
+    - bundle exec rake
+  expeditor:
+    executor:
+      docker:
+        image: ruby:2.7-stretch
```

Comment:
This isn't the right image name:

https://github.com/chef/ohai/blob/master/.expeditor/verify.pipeline.yml#L49